scripts/codebuild/install.sh

```diff
@@ -19,12 +19,15 @@ echo "${magenta}----- CUSTOMIZATIONS -------${reset}"
 ####  content/*
 
 ## get environment variable from parameter store.
+## setting stativ variables for now
+client="https://search-super-testy-search-test-xweemgolqgtta6mzqnuvc6ogbq.us-east-1.es.amazonaws.com"
```

Comment:
@rdoughty this will need to be reconciled with ryan's code.   Just touch base with him to make sure this will work.